### DISCUSSION TITLE
Show number of maximum users in addition to the number of current users

### DISCUSSION THREAD

**Discussion by wschell** [jamulussoftware/jamulus#1892] [UI Related]
I would like to have the information of the max allowed users in a jamulus session.

Background: I sometimes just listen to other musicians, but I don't want that others could not connect because I'm only listening. With this information I can see that the server get's full and disconnect.

I know I could remember what max number I saw on the connection dialog but to be honest - I forget it very fast...

**Comment by ann0see** [comment]
Maybe the client could save the max number of clients and append (full) after the server name?

**Reply by wschell** [reply]
If the client could save the number on connect I would prefer something like "10 of 14 musicians".
Don't know if it would work after connecting via ip or -c parameter. But it would be an improvment when connecting from a directory server list.

**Reply by gene96817** [reply]
How is this information different from what is shown in the Connection Setup Window?

**Reply by wschell** [reply]
It's the same - but I normally don't remember all the time the number of max users in a session.

**Reply by gene96817** [reply]
you can call up the Connection Setup window with cmd-C. I know it is not recommended, I just keep the Connection Setup window open.

**Reply by ann0see** [reply]
Or just use explorer.jamulus.io

**Reply by wschell** [reply]
Many ways lead to Rome - as we say. ;) But it would be nice to see it without workaround in the session. Just my opinion. 
Maybe someone want's to implement it one day.
